jest/state/index.ts

```diff
@@ -0,0 +1,277 @@
+import { layouts } from 'src/helpers/layout';
+// export function useSelector<TState = DefaultRootState, TSelected = unknown>(
+//     selector: (state: TState) => TSelected,
+//     equalityFn?: (left: TSelected, right: TSelected) => boolean
+// ): TSelected;
+
+import { Beneficiary, RootState } from 'src/modules';
+
+const defaultBeneficiary: Beneficiary = {
+    id: 0,
+    currency: '',
+    name: '',
+    state: '',
+    data: {
+        address: '',
+    },
+};
+
+const defaultUser = {
+    email: '',
+    level: 0,
+    otp: false,
+    role: '',
+    state: '',
+    uid: '',
+    profiles: [],
+    referal_uid: '',
+    labels: [],
+    phone: [],
+    created_at: '',
+    updated_at: '',
+};
+
+export const TEST_STATE: RootState = {
```

Comment:
This looks like this code brings to us 0 flexibility in modules. Even for tests we need small objects instead of a big one.

Reply:
There is no any modules. Once they are, there will be solution.